libs/cli/src/generators/ui/supported-ui-libraries.json

```diff
@@ -332,7 +334,8 @@
 			"@angular/core": "^18.0.0",
 			"@spartan-ng/ui-core": "0.0.1-alpha.353",
 			"class-variance-authority": "^0.6.0",
-			"clsx": "^1.2.1"
+			"clsx": "^1.2.1",
+			"@spartan-ng/ui-form-field-brain": "0.0.1-alpha.353"
```

Comment:
I think the select component should get this also?

Reply:
Sure, good catch!